src/sequence_batch.h

```diff
@@ -81,6 +81,25 @@ class SequenceBatch {
     return negative_sequence_batch_[sequence_index];
   }
 
+  // big_endian: N_pos is in the order of sequence
+  // little_endian: N_pos is in the order from the sequence right side to left, 
+  //                this is the order of the GenerateSeed
+  inline void GetSequenceNsAt(uint32_t sequence_index, bool little_endian, std::vector<int> &N_pos) {
```

Comment:
It seems that the second parameter is always set to true. Then why do we need that? It is easy to cause bugs with this kind of parameter. Better to remove it if it is unused.

Reply:
This is just for flexibility. In case we want the N in the order of the sequence in the future, which I think is more natural...
This is related to the next comment:
If the barcode is ACN, then the big_endian will return N at position 2, and little_endian will say N is at position 0. Little_endian is the order used in the GenerateSeedFromSequence function.